app/assets/components/messages/src/ui/components/messages-app/component.ts

```diff
@@ -0,0 +1,33 @@
+import Component, { tracked } from "@glimmer/component";
+declare const ADVOCATE;
+
+export default class Messages extends Component {
+  @tracked state = {
+    events: [],
+    events_empty: true
+  }
+
+  constructor(options) {
+    super(options);
+
+    this.load_default_events();
+    // periodically fetch events?
+
+    // GLOBAL
+    ADVOCATE.subscribe(this);
```

Comment:
Subscribe the component for updates. I should be more clear about the callbacks into the component we expect.